platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdminMBean.java

```diff
@@ -184,6 +184,17 @@ String createFactoryConfigurationForLocation(String factoryPid, String location)
     boolean update(String pid, Map<String, Object> configurationTable) throws IOException;
 
     /**
+     * Interceptor method for special handling of guest claims profile data.
+     *
+     * @param pid the persistent identifier of the configuration
+     * @param configurationTable the table of properties
+     * @throws IOException if the operation fails
+     * @see #update(String, Map) for the generic version of this method
+     */
+    boolean updateGuestClaimsProfile(String pid, Map<String, Object> configurationTable)
```

Comment:
Method signature should match the `update(String, Map)` method since we are passing the data along so the configs can get persisted.